src/main/java/mate/academy/rickandmorty/init/InitDataBase.java

```diff
@@ -0,0 +1,56 @@
+package mate.academy.rickandmorty.init;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.dto.CharacterResponseDto;
+import mate.academy.rickandmorty.exception.DataProcessException;
+import mate.academy.rickandmorty.mapper.DtoMapper;
+import mate.academy.rickandmorty.model.Character;
+import mate.academy.rickandmorty.repository.CharacterRepository;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class InitDataBase implements CommandLineRunner {
+    private static final String URL = "https://rickandmortyapi.com/api/character/";
+    private static final int NUMBER_OF_CHARACTERS = 20;
+    private final DtoMapper mapper;
+    private final ObjectMapper jasonMapper;
+    private final CharacterRepository characterRepository;
+
```

Comment:
not fixed previous comment 
need to create `HttpClient` bean in your Spring Configuration
```suggestion
private final HttpClient httpClient;

```

Reply:
This method runs only once when the application started, isn't it?